viite-UI/src/view/ProjectChangeTable.js

```diff
@@ -216,6 +216,24 @@
       return ((changeInfoSeq.reversed) ? '<td class="project-change-table-dimension">&#10004;</td>' : '<td class="project-change-table-dimension"></td>');
     }
 
+    function getAdministrativeClassText(administrativeClass) {
```

Comment:
Tähän joku "TODO: VIITE-2568, getAdministrativeClassText to be removed when backend side of AdministrativeClass addition completed." ?

Reply:
Ihan tällee sivumaininta että noi TODO unohtuu tonne helposti koodiin. Jos sellainen pitää laittaa mieluusti joku tiketin numero jossa muutos oikeasti tehdään ja tiketissä linkitys koodissa olevaan TODO kohtaan.

Reply:
Ei tuo funktio ole mun mielestä väliaikainen. Tuossa vaihtuu tuo changeInfoSeq.target.roadType -> changeInfoSeq.target.administrativeClass sitten kun bäkkäri on valmis. Backendistä tulee muutosten jälkeen edelleen numero arvo joka tulee muuttaa tekstiksi muutostauluun. 😮

Reply:
Jos TODO-sana kiusaa, voi tuon toki kertoa myös muodossa "Temporary roadType->administrativeClass transformation until backend supports administrativeClass" ilman TODOakin - ja vastaavasti tuonne ylemmäs. TODOn on kuitenkin tarkoitus imenomaan kertoa, että tähän tarttis vielä palata.
Funktioselitteet (miksi/mihin tätä käytetään/tarvitaan) ovat aina kannatettavia, ja samoin "epätyypillisten" koodin kohtien kommentit.
(Tässä tapauksessa Backend-muutostiketti on sama kuin tämä tiketti, ja bäkkäri on myös tiimillä työn alla. Sinänsä, jos bäkkärikin valmistuu ihan just, näiden ajatellut myöhemmät muutostarpeet muistetaan kyllä ilman kommenttejakin. Mutta Tällöinkin TODOt auttavat etsinnässä.)

Reply:
Hyym itse aattelisin että sitten kun koodi on pull request vaiheessa mitään TODO hommia ei enään tarvita, kun kaikki tarvittavat toimet on jos tehty. On ne ihan käteviä kehitys vaiheessa. Mut joo ihan periaate ja tapa asia.

Reply:
Samaa mieltä - aina kun tehdään kokonainen asia valmiiksi yhdessä pullarissa, eikä koodiin ole kyseisen fiksin/tiketin/featuren puitteissa enää tässä kehityksen vaiheessa tarkoitus palata.
(Mut nyt on kyl tän PR:n kannalta aika metaleveliä jo...)

Reply:
Jes, nyt on hienot, sekä tämä että tuo tuolla alla. :)